apps/studio/webpack.sanity.js

```diff
@@ -0,0 +1,64 @@
+const path = require("path");
```

Comment:
Dette var en fil jeg måtte legge til for å få Sanity til å støtte pre-ES2015 kode i node_modules (som react-aria har). Ref [supportsaken](https://sanity-io-land.slack.com/archives/C9Z7RC3V1/p1659614630387489)